setup.py

```diff
@@ -13,8 +13,8 @@ def read(fname):
 
 
 install_requires = [
-    'setuptools',
     'pbkdf2',
+    'mock',
```

Comment:
the install doesn't require mock, just the tests. You should install `mock` in the travis config.